RPS Project/Tests.py

```diff
@@ -0,0 +1,73 @@
+import random
+
```

Comment:
Please use the Python unittesting framework. https://docs.python.org/3/library/unittest.html

If this is too difficult to digest, please try this: http://pythontesting.net/framework/unittest/unittest-introduction/